src/fileiter.cpp

```diff
@@ -378,7 +396,11 @@ inline _fi_find_handle find_first_file(const char* wild,  _fi_find_data& data)
    if (::MultiByteToWideChar(CP_ACP, 0,  wild, wild_size,  wide_wild, wild_size + 1) == 0)
       return _fi_invalid_handle;
 
+# if BOOST_PLAT_WINDOWS_RUNTIME
+   return FindFirstFileExW(wide_wild, FindExInfoStandard, &data, FindExSearchNameMatch, NULL, 0);
```

Comment:
(thanks for the review: @MarcelRaad )

I will switch to unconditionally use FindFirstFileExW.

As it were boost.predef was being pulled in by something else which is the reason why it worked. I guess it is ok to add a dependency to boost.predef?